ios/Headless Universal Checkout/RNTPrimerHeadlessUniversalCheckoutCardFormUIManager.swift

```diff
@@ -0,0 +1,117 @@
+//
+//  RNTPrimerHeadlessUniversalCheckoutCardFormUIManager.swift
+//  primer-io-react-native
+//
+//  Created by Evangelos on 4/3/22.
+//
+
+import Foundation
+import PrimerSDK
+
+@objc(PrimerHeadlessUniversalCheckoutCardFormUIManager)
+class RNTPrimerHeadlessUniversalCheckoutCardFormUIManager: RCTViewManager {
+    
+    @objc
+    override static func requiresMainQueueSetup() -> Bool {
+        return true
+    }
+    
+    private lazy var cardFormUIManager: PrimerHeadlessUniversalCheckout.CardFormUIManager = {
+        return try! PrimerHeadlessUniversalCheckout.CardFormUIManager()
+    }()
+    private var successCallback: RCTResponseSenderBlock? {
+        didSet {
+            
+        }
+    }
+    
+    deinit {
+        print("🧨 deinit: \(self) \(Unmanaged.passUnretained(self).toOpaque())")
+    }
+        
+    override init() {
+        super.init()
+        do {
+            self.cardFormUIManager = try PrimerHeadlessUniversalCheckout.CardFormUIManager()
+            self.cardFormUIManager.cardFormUIManagerDelegate = self
+        } catch {
+            
+        }
+        
+        print("⭐ init: \(self) \(Unmanaged.passUnretained(self).toOpaque())")
+    }
+    
+    @objc
+    override func constantsToExport() -> [AnyHashable : Any]! {
+        return ["message": "Hello from native code"]
+    }
+    
+    func cardFormUIManager(_ cardFormUIManager: PrimerHeadlessUniversalCheckout.CardFormUIManager, isCardFormValid: Bool) {
+        
+    }
+        
+    @objc
+    func setInputElements(_ inputElements: String, errorCallback: RCTResponseSenderBlock, successCallback: @escaping RCTResponseSenderBlock) {
+        print("RNTPrimerHeadlessUniversalCheckoutCardFormUIManager.setInputElements\n\(inputElements)")
+        print("⭐ \(self) \(Unmanaged.passUnretained(self).toOpaque())")
+        self.successCallback = successCallback
+        self.successCallback?(["what??"])
```

Comment:
Same as above, it's not implemented yet, that's just testing the callback

Reply:
Oh cool! Perhaps something more explicit in this case just to avoid any issue / PR / comments from those wanting to play with the HUC 
```suggestion
        self.successCallback?(["Not implemented yet. This is a callback example parameter"])
```